kubernetes/assets/recommended_monitors/monitor_cluster_cpu.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "[kubernetes] Monitor Kubernetes Cluster CPU Usage",
+	"type": "query alert",
+	"query": "avg(last_5m):sum:kubernetes.cpu.requests{*} by {kubernetes_cluster} * 100 / sum:kubernetes.cpu.capacity{*} by {kubernetes_cluster} > 90",
```

Comment:
Notes:
Depends on the customer setup, this could be noisy.
Could monitor if pod is pending for a while instead. 
This could make sense for manual management of clusters, but not dynamic management.

Reply:
Could measure total cpu usage instead.

Reply:
Next steps: 
look at pending pods metrics.